bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/MadokaValue.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bluetooth.daikinmadoka.internal.model;
+
+import java.nio.ByteBuffer;
+
+/**
+ *
+ * @author blafois
+ *
+ */
+public class MadokaValue {
+
+    private int id;
+    private int size;
+    private byte[] rawValue;
```

Comment:
If this value can be null then you will have to add the following annotation.
```suggestion
    private byte @Nullable [] rawValue;
```